scripts/parse_prophecy.py

```diff
@@ -61,6 +56,26 @@ async def file_pointer_to_sample_map(
 
         return await super().file_pointer_to_sample_map(file_pointer, delimiter)
 
+    async def file_pointer_to_participant_map(
+        self,
+        file_pointer,
+        delimiter: str,
+    ) -> Dict[Any, Dict[Any, List[Any]]]:
+        """Parse manifest into a dict"""
+
+        # Skipping header metadata lines
+        for line in file_pointer:
+            if line.strip() == '""':
+                break
+
+        return await super().file_pointer_to_participant_map(file_pointer, delimiter)
+
+    def get_participant_id(self, row) -> str:
+        """Get external participant ID from row"""
+        if not self.participant_column or self.participant_column not in row:
+            raise ValueError('Participant column does not exist')
+        return row[self.participant_column][:-4]
```

Comment:
Why `[:-4]`?

Reply:
Right, to strip off. I think it's safer to instead parse that other file that we got from them, where we have both `Specimen ID` and `participantid` columns. Just in case the format changes and the `[:-4]` holds no longer.

Reply:
Removed :)